crates/sui-protocol-config/src/lib.rs

```diff
@@ -240,6 +240,16 @@ impl Default for Chain {
     }
 }
 
+impl Chain {
+    pub fn as_str(self) -> &'static str {
+        match self {
+            Chain::Mainnet => "mainnet",
+            Chain::Testnet => "testnet",
```

Comment:
no devnet?

Reply:
I believe this is for getting the chain id and we only store chain id for long running networks, i.e. mainnet and testnet.

Reply:
yep that's correct! Since devnet changes so frequently we can't know the chainid's for those instances without a bit of churn.

Reply:
Ah yes, forgot about that. Thanks!